requirements.txt

```diff
@@ -1,4 +1,4 @@
-numpy==1.8.0
-numpydoc==0.4
-networkx==1.8.1
+numpy>=1.8.0
+numpydoc>=0.4
+networkx>=1.8.1
```

Comment:
Should we not have a requirements.txt as all other add-ons? I know it's not used, but I see it as a nice statement to what the add-on needs for running properly. Can we include it somehow or should we indeed obliterate it?